pytests/func/test_threads.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root
+# for license information.
+
+from __future__ import print_function, with_statement, absolute_import
+import pytest
+from pytests.helpers.timeline import Event
+from pytests.helpers.session import DebugSession
+
+
+@pytest.mark.parametrize('args, count', [
+    ('single', 1),
+    ('multi', 3),
+])
+def test_multi_thread(pyfile, run_as, start_method, args, count):
```

Comment:
`count` is a dependent arg, so it should go into a dict.

Also, these names are a bit confusing at a glance, as in "multi-threaded single thread". :) Maybe "one" and "many"?